src/Cake.Parallel.Tests/ParallelGraphExtensionTests.cs

```diff
@@ -42,6 +42,18 @@ public void Throws_On_Circular_References()
             _sb.ToString().Length.ShouldBe(7);
         }
 
+        [Fact]
+        public async Task Should_Execute_DependenciesAndDependees_Asynchronously()
+        {
+            await _graph.Traverse("h", (nodeName, cts) =>
+            {
+                var task = (ActionTask)_tasks.First(_ => _.Name == nodeName);
+                task.Actions.ForEach(action => action(null));
+                return Task.CompletedTask;
+            }).ConfigureAwait(false);
+            _sb.ToString().Length.ShouldBe(3);
```

Comment:
Would it be worth it to change the assertion here to `_sb.ToString().Equals("h2h")`? It does take a little bit of parsing to figure out that `h1` does not get executed, so the `.Length.ShouldBe(3)` could be a little misleading.

Reply:
No problem. I will make the changes and issue another PR. I will also downgrade the tools versions. I upgraded it just to keep it up-to-date with the latest cake.